app.rb

```diff
@@ -1,7 +1,16 @@
 require "sinatra/base"
-
+require "pg"
 class AbodenB < Sinatra::Base
   get "/" do
     "Welcome"
   end
+  get "/signup" do
+    erb :signup
+  end
+  post "/signup" do
+    redirect "/profile"
```

Comment:
Not calling the User.create method here - when feature testing the website manually, it isn't adding the signup details to the database

Reply:
resolved